resources/docs/database-backends.md

```diff
@@ -0,0 +1,64 @@
+# Database Backends
+
+RSS Guard has two database engines available for data storage
+
+- [SQLite](https://www.sqlite.org/index.html)
+- [MariaDB](https://mariadb.org/) (MySQL)
+
+You can manually choose which backend RSS Guard will use by changing the settings manually:
+
+1. Go to **Tools → Settings**
+2. Select **Data Storage**
+
+You have the option of 
+
+## SQLite
+
+The SQLite database engine is the one enabled by default as it provides a solid, proven and reliable way of storing database information that is easy to backup manually or via the use of file sync applications like Nextcloud, Dropbox, iCloud or OneDrive.
```

Comment:
This is great, merge this carefully with already existing DB backends stuff: https://github.com/martinrotter/rssguard/blob/master/resources/docs/Documentation.md#database-backends

It is generally for people who start to do PRs to complement existing chapters with more info.... anyway good job, just needs some polishing.